roles/monitoring/files/etc_monit_conf.d_pgsql

```diff
@@ -1,4 +1,4 @@
-check process postgres with pidfile /var/run/postgresql/9.1-main.pid
+check process postgres with pidfile /var/run/postgresql/9.4-main.pid
```

Comment:
On Ubuntu Xenial, both PostgreSQL 9.4 and 9.5 are available. I suppose we stick to 9.4 to improve compatability with Debian Jessie